src/components/TeamBio.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import Image from 'react-bootstrap/Image'
+
+class TeamBio extends React.Component {
+    render() {
+        const { pic, name, role, bio } = this.props.tmData;
+
+        return (
+            <>
+                <Image src={ pic } fluid></Image>
+                <p>{ name }</p>
+                <p>{ role }</p>
```

Comment:
Changing this to `h4 className="fs-6 mb-4"` does the same as the previous, but a bit smaller.